website/docs/security/authentication.md

```diff
@@ -104,3 +110,102 @@ If you encounter any issues while authenticating with FerretDB, try setting the
 Please refer to your specific driver documentation on how to set this field.
 
 If this does not resolve your issue please file a bug report [here](https://github.com/FerretDB/FerretDB/issues/new?assignees=ferretdb-bot&labels=code%2Fbug%2Cnot+ready&projects=&template=bug.yml).
+
+## Experimental authentication mode
+
+FerretDB provides a new experimental authentication mode that supports the `SCRAM-SHA-1` and `SCRAM-SHA-256` authentication mechanisms.
+This mode enables FerretDB to manage users by itself and also provide support for more user management commands.
+
+You can enable this mode by setting the `FERRETDB_TEST_ENABLE_NEW_AUTH` flag or `--test-enable-new-auth` to `true`.
```

Comment:
We probably need to link to flags.md somewhere around there. We also should update flags.md (despite it saying "Do not document `--test-XXX` flags here" – that is an exception)